ChangeLog

```diff
@@ -1,3 +1,14 @@
+2016-06-17  Daniel Standage <daniel.standage@gmail.com>
+
+   * scripts/filter-abund-single.py: add -o/--outfile option.
+   * tests/test_script_output.py: mov function to calculate file MD5 hashes to
+   test utils module (tests/khmer_tst_utils.py).
+   * tests/test_filter_abund.py: add minimal test for new -o option.
```

Comment:
seems like we should move all the test_filter_abund code from test_scripts into this file now, no?

alternatively, don't create a new file and just throw the test into test_scripts.  But that's getting large. Hmm.